airflow/example_dags/example_http_operator.py

```diff
@@ -81,7 +81,7 @@
     task_id='http_sensor_check',
     http_conn_id='http_default',
     endpoint='',
-    params={},
```

Comment:
Nit: Can we call these request_params or something instead? I think this is a more logical name.